source/agora/network/Client.d

```diff
@@ -146,6 +146,9 @@ public class NetworkClient
     /// Logger uniquely identifying this client
     private Logger log;
 
+    /// The identity associated with this node - may be empty
+    private Identity identity_;
```

Comment:
Can you explain the meaning of adding `_` in the name other than other member variables?

Reply:
Because otherwise it would conflict with the method that is named `identity`.